warehouse/macros/define_gtfs_guidelines.sql

```diff
@@ -0,0 +1,25 @@
+-- declare checks
```

Comment:
you could use https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-variables instead of macros potentially

Reply:
Is there a specific reason you think that would be better? I thought that macros were more accessible (I feel like we don't want average dbt users editing `dbt_project.yml` a lot? And this table is going to have a lot of iteration.)

Reply:
IMO I think a macro for a string value is overkill, that's about it

Reply:
Discussed offline -- my summary is: All the `check` values will be used at least two places (in their actual staging check table and in the construction of the index), I wanted to make it easy to be able to reference these hard-coded values if we do want to use them other places / make it so we can update one location and be confident it will propagate